### PR TITLE
ADMINUI-2324 AdminUI Needs to provide a Warning when enabling/disabling Global FW Rules

### PR DESCRIPTION
**Pull request by Arc311**

### CODE REVIEW

File: www/js/components/pages/vm/fwrules-list.js [hunk 1]
```diff
@@ -109,8 +154,8 @@ var FWRulesList = require('../../../views/collection').extend({
         }, this);
 
         var self = this;
-        this.on('itemview:delete:rule', function(iv) {
-            $.delete_(iv.model.url(), function(data) {
+        this.on('itemview:delete:rule', function (iv) {
```
Comment (andyYatskov) [inline www/js/components/pages/vm/fwrules-list.js]: iv -> rule

File: www/js/components/pages/vm/global-rule-warning.hbs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="modal-body">
+            <p class="alert alert-warning">
+                You are attempting to {{action}} a Global firewall rule.
+                This action will immediately affect ALL VM's with firewall enable in this data centre.
+                Are you sure you want to do this?
+            </p>
+        </div>
+        <div class="modal-footer">
+            <button class="btn" data-dismiss="modal">No</button>
+            <button href="#" class="btn btn-danger" data-dismiss="modal" type="submit">Yes</button>
```
Comment (andyYatskov) [inline www/js/components/pages/vm/global-rule-warning.hbs]: delete attribute href

File: www/js/components/pages/vm/fwrules-list.js [hunk 3]
```diff
@@ -74,18 +119,18 @@ var FWRulesList = require('../../../views/collection').extend({
...
-            this.collection = new FWRules(null, {params: { owner_uuid: options.user }});
+            this.collection = new FWRules(null, {params: {owner_uuid: options.user}});
         } else {
             this.collection = new FWRules();
         }
 
-        this.on('itemview:disable:rule', function(iv) {
-            iv.model.on('sync', function() {
+        this.on('itemview:disable:rule', function (iv) {
+            iv.model.on('sync', function () {
```
Comment (andyYatskov) [inline www/js/components/pages/vm/fwrules-list.js]: please rename: iv -> rule
and below

File: www/js/components/pages/vm/fwrules-list.js [hunk 4]
```diff
@@ -74,18 +119,18 @@ var FWRulesList = require('../../../views/collection').extend({
...
                 app.vent.trigger('notification', {
                     level: 'success',
                     message: "Firewall rule disabled successfully."
```
Comment (andyYatskov) [inline www/js/components/pages/vm/fwrules-list.js]: use single quotes everywhere

File: www/js/components/pages/vm/fwrules-list.js [hunk 5]
```diff
@@ -74,18 +119,18 @@ var FWRulesList = require('../../../views/collection').extend({
...
+            this.collection = new FWRules(null, {params: {owner_uuid: options.user}});
         } else {
             this.collection = new FWRules();
         }
 
-        this.on('itemview:disable:rule', function(iv) {
-            iv.model.on('sync', function() {
+        this.on('itemview:disable:rule', function (iv) {
+            iv.model.on('sync', function () {
                 app.vent.trigger('notification', {
```
Comment (andyYatskov) [inline www/js/components/pages/vm/fwrules-list.js]: DRY 
create common function for notifications

File: www/js/components/pages/vm/fwrules-list.js [hunk 6]
```diff
@@ -109,8 +154,8 @@ var FWRulesList = require('../../../views/collection').extend({
         }, this);
```
Comment (andyYatskov) [inline www/js/components/pages/vm/fwrules-list.js]: what happens if rule was not updated?

**Comment by andyYatskov** [comment]
🔢

File: www/js/components/pages/vm/global-rule-warning.hbs [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+<div class="modal-dialog">
+    <div class="modal-content">
+        <div class="modal-header">
+            <h3 class="modal-title">Warning: {{titleAction}} global firewall</h3>
+        </div>
+        <div class="modal-body">
+            <p class="alert alert-warning">
+                You are attempting to {{action}} a Global firewall rule.
+                This action will immediately affect ALL VM's with firewall enable in this data centre.
```
Comment (dan-luk) [inline www/js/components/pages/vm/global-rule-warning.hbs]: "firewall enabled"

Comment (dan-luk) [inline www/js/components/pages/vm/global-rule-warning.hbs]: This is not new line, is it ok?

Comment (Arc311) [inline www/js/components/pages/vm/global-rule-warning.hbs]: It's ok

**Comment by dan-luk** [comment]
🔢

**Comment by dan-luk** [comment]
🔢

**Comment by dan-luk** [comment]
🔢

**Pull request closed without merging**